src/pages/Detail/Detail.jsx

```diff
@@ -1,7 +1,141 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import variables from '../../styles/variables';
+import DetailNav from './component/DetailNav';
+import DetailIntro from './component/DetailIntro';
+import DetailAside from './component/DetailAside';
 
-function Detail(props) {
-  return <div>Detail</div>;
+function Detail() {
+  const [lecture, setLecture] = useState({});
+  const [isLogin, setIsLogin] = useState(false);
+  // const [subscribe, setSubscribe] = useState(false);
+  const { coverImage, coverGallery, review } = lecture;
+
+  useEffect(() => {
+    fetch(`/data/data_detail.json`)
+      .then(response => response.json())
+      .then(result => {
+        if (result.data) {
+          setLecture(result.data);
+        }
+      });
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    fetch(`/data/data_detail.json`)
      .then(response => response.json())
      .then(result => setLecture(result.data));
  }, []);
```

조건문 없이 적용해도 정상 동작을 합니다!